src/dropdown/dropdown.ts

```diff
@@ -55,6 +82,12 @@ export class NgbDropdown {
    */
   open(): void {
     if (!this._open) {
+      if (this.container === 'body') {
+        let container = window.document.querySelector(this.container);
+        this._renderer.attachViewAfter(container, this._dropdownMenu.elementRef.nativeElement);
```

Comment:
This will append the dropdown-menu AFTER body. This works for me with: `container.lastElementChild` but I don't know much about browser-support.
2nd parameter of attachViewAfter should be an array as well:
`this._renderer.attachViewAfter(container, [this._dropdownMenu.elementRef.nativeElement]);`